workspace.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+#
+#copyright agloo in 2016 under the WTFPL 
+#constructs and groups windows into workspaces to switch between using your favorite keybinder.
+
+NUM_WS=9
+
+help() {
+	cat << EOF
+usage: $(basename $0) [-hinp] [-g ws_num] [-m ws_num] 
+	-h: Displays this message
+	-i: Initializes workspaces and puts you in workspace 0. Should be called once in a startup script.
+	-n: Move up one workspace
+	-p: Move down one workspace
+	-g, --ws_num: go to the workspace specified by ws_num
+	-m, --ws_num: move the currently focused window to the worskpace specified by ws_num
+EOF
+	exit 1
+}
+#initializes us in workspace 0
+ws_init() {
+	mkdir /tmp/workspaces/
+	for i in {0..$NUM_WS}
+	do
+		touch /tmp/workspaces/ws"$i"
```

Comment:
While this would work, I'd suggest using `:> /tmp/workspaces/ws"$i"` to create an empty file, rather than `touch`. Just a semantic change though, Feel free to ignore the comment.